test/integration/targets/openssl_csr/tasks/CentOS.yml

```diff
@@ -0,0 +1,18 @@
+- name: Install epel
+  become: yes
+  package:
+    name: epel-release
+    state: latest
+
+- name: Install python-pip
+  become: yes
+  package:
+    name: python-pip
+    state: latest
+
+- name: pip install PyOpenSSL
+  become: yes
+  package:
+    name: PyOpenSSL
+    state: latest
+    use: pip
```

Comment:
Install using the pip module instead of the package module.

The same is true for the other tests.